element_electrode_localization/electrode_localization.py

```diff
@@ -24,24 +24,14 @@ def activate(
     create_tables=True,
     linking_module=None,
 ):
-    """
-    activate(electrode_localization_schema_name, coordinate_framework_schema_name=None,
-             *, create_schema=True, create_tables=True, linking_module=None)
-        :param electrode_localization_schema_name: schema name on the database server to
-                                           activate the `electrode_localization` element
-        :param coordinate_framework_schema_name: schema name on the database server to
-                                            activate the `coordinate_framework` element.
-                                            May be omitted if already activated
-        :param create_schema: when True (default), create schema in the database if it
-                              does not yet exist.
-        :param create_tables: when True (default), create tables in the database if they
-                              do not yet exist.
-        :param linking_module: a module name or a module containing the
-         required dependencies to activate the `electrode_localization` element:
-            Upstream tables:
-                + ProbeInsertion: table referenced by ElectrodePosition, typically
-                                  identifying a Probe Insertion instance
-                + probe: the probe schema - from element-array-ephys
+    """Activates the `electrode_localization` and `coordinate_framework` schemas. 
+
+    Args:
+        electrode_localization_schema_name (str): A string containing the name of the electrode_localization schema.
```

Comment:
please wrap to 88 where possible. For long arg descriptions, we can just increase the indent on the next line, not wrap to the `:`
```suggestion
        electrode_localization_schema_name (str): A string containing the name of the 
            electrode_localization schema.
```

Reply:
Should be updated (but I'm still not sure if its wrapping correctly)

Reply:
It doesn't seem to be updated, but you can do so by committing the suggestion above